VKClientRM/VKClientRM/Hellper/VKNetworkService.swift

```diff
@@ -67,65 +71,108 @@ final class VKNetworkService {
         return parameters
     }()
 
+    private lazy var parametersUsersGetVK: Parameters = {
+        var parameters = generalParameters
+        parameters[Constants.fieldsText] = Constants.photoText
+        return parameters
+    }()
 
     // MARK: - Public Methods
 
-    func fetchFriendsVK(completion: @escaping ([ItemPerson]) -> Void) {
-        let path = Constants.methodText + Constants.friendsGetText
+    func fetchFriendsVK(completion: @escaping (Result<[ItemPerson], Error>) -> ()) {
+        let path = "\(Constants.methodText)\(Constants.friendsGetText)"
         let url = "\(Constants.baseUrl)\(path)"
-        Alamofire.request(url, method: .get, parameters: parametersFriendsVK).responseData { response in
-            guard
-                let self = self,
-                let data = response.value,
-                let items = try? JSONDecoder().decode(Person.self, from: data).response.items
-            else { return }
-            completion(items)
+        AF.request(url, method: .get, parameters: parametersFriendsVK).responseData { response in
+            guard let data = response.value else { return }
+            do {
+                let items = try JSONDecoder().decode(Person.self, from: data).response.items
+                completion(.success(items))
+            } catch {
+                completion(.failure(error))
+            }
         }
     }
 
-    func fetchPhotosVK(userID: String, completion: @escaping ([String]) -> Void) {
-        let path = Constants.methodText + Constants.photosGetAllText
+    func fetchPhotosVK(person: ItemPerson, completion: @escaping (Result<ItemPerson, Error>) -> ()) {
+        let path = "\(Constants.methodText)\(Constants.photosGetAllText)"
         let url = "\(Constants.baseUrl)\(path)"
         var parametersPhotos = generalParameters
-        parametersPhotos[Constants.ownerIdText] = userID
-        Alamofire.request(url, method: .get, parameters: parametersPhotos).responseData {  response in
-            guard
-                let self = self,
-                let data = response.value,
-                let items = try? JSONDecoder().decode(Photo.self, from: data).response.items
-            else { return }
-            var photosURLText: [String] = []
-            for item in items {
-                photosURLText.append(item.url)
+        parametersPhotos[Constants.ownerIdText] = "\(person.id)"
+        AF.request(url, method: .get, parameters: parametersPhotos).responseData { response in
+            guard let data = response.value else { return }
+            do {
+                let items = try JSONDecoder().decode(Photo.self, from: data).response.items
+                let updatePerson = person
+                let photosPerson = List<ItemPhoto>()
```

Comment:
```suggestion
                let photosPerson = List<ItemPhoto>()
```
почему такое написание?